pandas/core/window.py

```diff
@@ -305,10 +306,10 @@ def _center_window(self, result, window):
                 result = np.copy(result[tuple(lead_indexer)])
         return result
 
-    def aggregate(self, arg, *args, **kwargs):
-        result, how = self._aggregate(arg, *args, **kwargs)
+    def aggregate(self, func, *args, **kwargs):
```

Comment:
This is nice - I did something similar here:

https://github.com/pandas-dev/pandas/pull/26089/files#r275180397

Similarly I think we should add a whatsnew here though to be conservative from an API perspective